src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  sourceString.replace(/;\s*;/g, ' ').trim();
+
+  const styleRules = sourceString.split(';');
+
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider naming the variable to reflect its contents, such as `styleObject`.